package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "dnd-8th-7-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "postinstall": "husky install",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "dependencies": {
+    "@next/font": "13.1.6",
+    "@types/node": "18.11.19",
+    "@types/react": "18.0.27",
+    "@types/react-dom": "18.0.10",
+    "axios": "^1.3.2",
+    "next": "13.1.6",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "typescript": "4.9.5",
+    "zustand": "^4.3.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.12",
+    "@commitlint/cli": "^17.4.2",
+    "@commitlint/config-conventional": "^17.4.2",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-essentials": "^6.5.16",
+    "@storybook/addon-interactions": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
+    "@storybook/addon-postcss": "^2.0.0",
+    "@storybook/builder-webpack5": "^6.5.16",
+    "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/react": "^6.5.16",
+    "@storybook/testing-library": "^0.0.13",
+    "@typescript-eslint/eslint-plugin": "^5.50.0",
+    "autoprefixer": "^10.4.13",
+    "babel-loader": "^8.3.0",
+    "eslint": "^8.33.0",
+    "eslint-config-next": "^13.1.6",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-storybook": "^0.6.10",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.1.0",
+    "postcss": "^8.4.21",
+    "prettier": "^2.8.3",
+    "tailwindcss": "^3.2.4"
+  },
+  "lint-staged": {
+    "**/*.(js|jsx|ts|tsx)": [
+      "eslint --cache --fix",
+      "yarn run prettier --write"
+    ]
```

허스키를 직접 써보지는 못했는데, 협업하는데 좋은 툴이네요!
하나 더 제안 드리자면, 타입 체킹을 추가하는 건 어떨까요??

```suggestion
    "**/*.(js|jsx|ts|tsx)": [
      "eslint --cache --fix",
      "yarn run prettier --write",
      "tsc --noEmit"
    ]
```

다만 이 과정을 추가하면 트랜스파일링하는 데에 시간이 조금 더 소요된다는 점이 있을 수 있을 것 같네요..!

---

넵 좋아요! 그래도 아직은 큰 차이 없는 것 같아서 추가했습니다 [db44b63](https://github.com/dnd-side-project/dnd-8th-7-frontend/pull/2/commits/db44b63b16252f828ca9136982de774db15526f8)
`tsc --noEmit`터미널로 직접 입력하면 정상 작동하는데, husky로 추가하면 왜 인지 에러가 나더라구요..
그래서 bash -c 옵션을 사용했습니다